README.md

```diff
@@ -12,25 +12,34 @@ Video: https://youtu.be/KiDnPsnLmLU
 
 # Notes/Howto
 
-To run demo with four cones use:
+There has been serious development to extend original code suited mainly for
+modified John Deere X300R garden tractor to more general tool. Now it runs
+on more than four machines, including autonomous boat. It provides logging
+and replay facility, so simple logging of multiple inputs is unified and
+stored in binary format with timestamps.
```

Comment:
I don't think anyone cares about what the original code did (whatever original code is). I think we should say why we decided to write it and why other approaches didn't work for us. For me it would be the logging facility coupled to the replay facility allowing for record & replay. It answers the question "why aren't you just using ROS" in a fairly direct way.

Reply:
Oh, I see. Let's move the obsolete code to the obsolete directory and we don't have to explain what the code did originally.

Reply:
Maybe we should also add a sentence or two about the architecture - we have modules that have input ports and output ports, the connections are described in the config file and these ports are logged. And that is what enables record and replay. Robot is set of modules, module is instance of a driver, init arguments are also in config file.